### ISSUE TITLE
Playing ayah feature Unusable on chrome

### ISSUE THREAD

**Issue by hannanmumtaz** [quran/quran.com-frontend#1074]
**I'm submitting a ...**

- [ X] bug report
- [ ] feature request
- [ ] other

**Current behavior:**

Hi , First of all great work. JazakAllahKhair

Am unable to use the site on chrome , which clicking on ayahs  some of them work and some don't , irrespective to how many times i refresh.
I also noticed in console and getting a too many recirects error on maniifest.json

Also when i click on Play on certain verse no network call is initiated. hence no mp3 files is loaded and nothing palys.

https://quran.com/2/130-141

In the above link i tried multiple times refreshing and clearing cookies but am unable to play verse 140.

**Expected / new behavior:**

Clicking on a verse should play the verse.
**For issue, please provide steps to reproduce it:**
- Open Chrome version 71
- Goto https://quran.com/2/130-141
- Scroll to verse 140
- Click play 
**Anything else:**

Chrome Version : 71.0.3578.98
![image](https://user-images.githubusercontent.com/13946144/52891820-96dd5100-31e1-11e9-9678-f6ee6f2152b3.png)

The above works in edge , issue seems to be related only to chrome.

**Comment by nasir-scalestack** [comment]
This is a common issue for me as well when I use the website on chrome.

**Comment by dilan-omer** [comment]
is it only chrome, or is it the same on other browsers?

**Comment by abdurrahmaan** [comment]
Chrome version 73 seems to have fixed it. The verse plays for me

**Comment by codeshifu** [comment]
@abdurrahmaan not quite. I have the latest version of chrome and the issue persists. For me, I notice when I first load the page and play an ayah, it works fine. When an ayah is currently playing and I navigate to another page (chapter), I can seem to play any of the ayah from that page I just navigated to.

**Comment by dilan-omer** [comment]
ive been using chrome, and the issue gets fixed when i reload the page or specify the number of ayahs i want to be played in the surah, eg: quran.com/2/5-25. ive also noticed that it says its loading the ayah but there isnt any network calls to get the ayah's file, not sure why though

**Comment by carlosdelfino** [comment]
I am having similar problems in larger surah, I believe in larger than 10 ayat
